config/sync/core.extension.yml

```diff
@@ -19,6 +19,8 @@ module:
   ctools: 0
   datetime: 0
   dblog: 0
+  devel: 0
+  devel_generate: 0
```

Comment:
Since devel is a dev dependency this will cause issues when Drupal tries to install it on the prod site, but cannot find the devel codebase. I suggest uninstalling it.